marketplace/netlify-build/src/target-state.js

```diff
@@ -0,0 +1,19 @@
+import { get } from 'lodash-es';
+
+export function currentStateToEnabledContentTypes(currentState) {
+  const eiState = get(currentState, ['EditorInterface'], {});
+  const ctIds = Object.keys(eiState);
+
+  return ctIds.filter(id => !!get(eiState, [id, 'sidebar']));
+}
+
+export function enabledContentTypesToTargetState(contentTypes, enabledContentTypes) {
+  return {
+    EditorInterface: contentTypes.reduce((acc, [ctId]) => {
+      return {
+        ...acc,
+        [ctId]: enabledContentTypes.includes(ctId) ? { sidebar: { position: 2 } } : {}
```

Comment:
Should we be hardcoding position 2 here? Maybe this should be an installation option so it doesn't mess with peoples' sidebars if they don't want.

Reply:
This would be a good follow up but let's keep it as it is right now.